pkg/daemon/gateway_linux.go

```diff
@@ -480,6 +481,9 @@ func (c *Controller) setIptables() error {
 		protocols[0] = c.protocol
 	}
 
+	execer := k8sexec.New()
```

Comment:
we should move this to the init stage as theses values are seldom changed and exec is a heavy operation

Reply:
fixed

Reply:
The `setIptables` wiill still be called periodically, we should move it to the controller.initRuntime

Reply:
fixed